### ISSUE TITLE
Can't trigger paste event when clicking 'Paste' menu on paste dialog

### ISSUE THREAD

**Issue by xuchao777** [ckeditor/ckeditor-dev#2490]
## Type of report

Bug

## Provide detailed reproduction steps (if any)

1. On android, open the paste dialog then long press
2. It will show 'Paste' button, then click 'Paste' button
3. Can't trigger paste event (I want to make some changes in the paste event)
<img width="349" alt="2018-10-17 9 39 34" src="https://user-images.githubusercontent.com/10975431/47090345-5252a080-d255-11e8-984b-6aa69e6d59d4.png">

### Expected result
can trigger paste event

### Actual result
can't trigger paste event

## Other details

* OS: Android 6.0
* CKEditor version: 4.10
* Installed CKEditor plugins: …

**Comment by engineering-this** [comment]
Hi,

I tested it with Android 9.0 and latest CKeditor (4.10.1) everything works ok. Can you check if this is still valid issue for you with latest CKeditor?

You can try sample:
https://codepen.io/anon/pen/MPBOoR

Anything you paste should be transformed into `Test paste!`.

If you can't reproduce issue in given sample, but it is still valid in 4.10.1 with your configuration, please provide similar example with minimal config, so we can find out what is causing this bug.

**Comment by jacekbogdanski** [comment]
Hi, It's been a while since we last heard from you. We're closing this issue for now. Still, feel free provide us requested feedback, so that we can reopen this issue.

**Comment by jacekbogdanski** [comment]
I'm not sure if I understood you correctly the first time. About empty editor - do you mean that the editor is empty, or dialog which shows up after pressing the paste button?

I can reproduce the issue with empty dialog and would like to make sure if I understand the issue correctly. I'm wondering if pressing paste button with empty dialog shouldn't fire `paste` event because natively if you paste anything in the browser with empty clipboard it executes paste event.

**Comment by xuchao77** [comment]
Sorry, I mean the 'Paste' button in the empty dialog. I think any type of paste should executes paste event. Could you please help to fix it ?

**Comment by jacekbogdanski** [comment]
I agree it should behave as close to the native clipboard as possible.

**Comment by ItsNoHax** [comment]
I found that it's related to the Clipboard plugin which is neccesary. I do exactly the same thing using a CKEditor in a webview inside React-Native app. Thing is to remove the toolbar you need to remove the clipboard plugin and removing the clipboard plugin creates the behavior you are seeing.
